dlpar/dlpar_main.py.data/dlpar.yaml

```diff
@@ -1,3 +1,5 @@
+rpm_link: 'http://ausgsa.ibm.com//gsa/ausgsa/projects/h/htx/public_html/htxonly/'
```

Comment:
remove this address and please do not share IBM confidential  info in GitHub forum